Web Development/Drag-N-Drop/script.js

```diff
@@ -0,0 +1,39 @@
+const fill = document.querySelector('.fill')
+const empties = document.querySelectorAll('.empty')
+
+fill.addEventListener('dragstart', dragStart)
+fill.addEventListener('dragend', dragEnd)
+
+for(const empty of empties) {
+    empty.addEventListener('dragover', dragOver)
```

Comment:
Same as https://github.com/bradtraversy/50projects50days/blob/master/drag-n-drop/script.js

Reply:
Yup, but i changed the ui.